materialize-snowflake/sqlgen.go

```diff
@@ -207,7 +207,7 @@ SELECT {{ $.Table.Binding }}, {{ $.Table.Identifier }}.{{ $.Table.Document.Ident
 		{{- if $ind }}, {{ end -}}
 		$1[{{$ind}}] AS {{$key.Identifier -}}
 		{{- end }}
-		FROM @flow_v1/{{ $.RandomUUID }}
+		FROM @flow_v1/%s
```

Comment:
There's still the `RenderTableWithRandomUUIDTemplate` that is being used, although now it doesn't do anything with the UUID provided to it. As an alternative to using a Go template to create a string with a `%s` placeholder that is then used in `fmt.Sprintf`, I think it would be more clear to make `renderWithDir` more like the old `RenderTableWithRandomUUIDTemplate`, but instead of a UUID have the value inserted here be the `dir`. Or at least, clean up the `RenderTableWithRandomUUIDTemplate` function and its invocations.